lib/auth/resetpasswordtoken_test.go

```diff
@@ -163,6 +164,94 @@ func (s *ResetPasswordTokenTest) TestCreateResetPasswordTokenErrors(c *C) {
 
 	for _, tc := range testCases {
 		_, err := s.a.CreateResetPasswordToken(context.TODO(), tc.req)
-		c.Assert(err, NotNil, Commentf("test case %q", tc.desc))
+		c.Assert(err, check.NotNil, check.Commentf("test case %q", tc.desc))
 	}
 }
+
+// TestFormatAccountName makes sure that the OTP account name fallback values
+// are correct.
+func (s *ResetPasswordTokenTest) TestFormatAccountName(c *check.C) {
+	var tests = []struct {
```

Comment:
nit: `tests := []struct{...`
prefer to only use `var` for package-level variables or empty local variables.